gem/tests/test_auth.py

```diff
@@ -441,3 +447,289 @@ def test_auth_middleware(self):
     #     self.assertContains(response, "Welcome to the GEM Wagtail CMS")
     #     # test that the correct logout button is in in the template
     #     self.assertContains(response, "oidc/logout")
+
+
+class TestAllAuth(GemTestCaseMixin, TestCase):
+    def setUp(self):
+        self.user = get_user_model().objects.create_superuser(
+            username='superuser', email='superuser@email.com', password='pass')
+        self.main = self.mk_main(
+            title='main1', slug='main1',
+            path='00010002', url_path='/main1/'
+        )
+        self.site = self.main.get_site()
+
+    @override_settings(ENABLE_ALL_AUTH=True)
+    def test_admin_login_view(self):
+        res = self.client.get(reverse('wagtailadmin_login'))
+        self.assertEqual(res.status_code, 200)
+
+    def test_admin_login_view_default_allauth_setting(self):
+        res = self.client.get(reverse('wagtailadmin_login'))
+        self.assertEqual(res.status_code, 200)
```

Comment:
What is this test and the one above it actually supposed to be testing? You didn't make any changes to the `wagtailadmin_login` url
I presume this test and the one above it are supposed to load different pages but these tests don't make that clear. Please add a check to each of them to show they are loading the correct templates or content